triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -41,7 +41,7 @@ abstract class AbstractFuncTests
 
   this.getClass.getSimpleName can {
     "Failure testing" should {
-      val contractPairings = 500
+      val contractPairings = 400
```

Comment:
Looking at failing Windows test logging, I believe I'm observing only ~940 contracts being created on windows systems within a 5 minute test window. Reducing the number of contract pairings to 400 should hopefully allow these tests to run (without being flaky) in both Windows and *nix environments?